### DISCUSSION TITLE
Add database has error

### DISCUSSION THREAD

**Discussion by ccczhouxin** [dbeaver/dbeaver#20702] [Q&A] [answered]
I add a new database has these erros. the cloudbeaver version is 23.1.0 in windows10 ,when i add my database in feature.xml,it has some errors.
![image](https://github.com/dbeaver/cloudbeaver/assets/43168824/106f1bd3-d26e-47f2-ac27-b23b74aba608)

[error.txt](https://github.com/dbeaver/cloudbeaver/files/11858495/error.txt)

**Comment by flambchap** [comment]
Hi @ccczhouxin. Can you please tell me what database driver you're trying to add and to which DBeaver product?

**Comment by ccczhouxin** [comment]
> Can you please tell me what database driver you're trying to add and to which DBeaver product?

The database is developed by the company itself，dbeaver is 23.1.1,

**Comment by flambchap** [comment]
In the log file you send, it's clear that you're trying to run CloudBeaver. I suggest importing the dbeaver project in Eclipse first: https://github.com/dbeaver/dbeaver/wiki/Develop-in-Eclipse. After that, if you'd like to continue with IntelliJ IDEA, please refer to this article: https://github.com/dbeaver/dbeaver/wiki/Develop-in-IDEA.

**Comment by E1izabeth** [comment, marked as answer]
According to the logs it looks like you're trying to set promoted driver property to "true" while it accepts only integer values.
```
java.lang.NumberFormatException: For input string: "true"
        at java.base/jdk.internal.math.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2054)
        at java.base/jdk.internal.math.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
        at java.base/java.lang.Double.parseDouble(Double.java:651)
        at org.jkiss.utils.CommonUtils.toInt(CommonUtils.java:385)
        at org.jkiss.dbeaver.registry.driver.DriverDescriptor.<init>(DriverDescriptor.java:356)
```
